src/Restaurant.java

```diff
@@ -14,46 +14,36 @@
 	public static void setOrderList(Order[] orderList) {
 		Restaurant.orderList = orderList;
 	}
-
-	public int getCounter() {
-		return counter;
+	
+	public Customer[] getCustomerList() {
+		return CustomerList;
 	}
 
-	public void setCounter(int counter) {
-		this.counter = counter;
+	public void setCustomerList(Customer customerList[]) {
+		CustomerList = customerList;
 	}
 	
 	/**
 	 * this function will show menu to the customer
-	 * @throws IOException 
 	 */
-	public void showMenu() throws IOException {
-		System.out.println("Please select from follwing items by entering their Id");
-		for(int i=0; i < Menu.itemMenu.length; i++) {
-			System.out.println(Menu.itemMenu[i].getItemId() + " - " + Menu.itemMenu[i].getItemName() + " - " + Menu.itemMenu[i].getItemPrepTime());;
+	public void showMenu() {
+		try {
+			Iterator<Item> i = Menu.itemMenu.iterator();
+			while (i.hasNext()) {
+				Item item = (Item) i.next();
```

Comment:
Remove this unnecessary cast to "Item".
Since Iterator is already<Item> type casting is not required here.So change it to :

```suggestion
				Item item = i.next();
```